### DISCUSSION TITLE
Next.js ecommerce localization

### DISCUSSION THREAD

**Discussion by MohanLiSilk** [vercel/next.js#37733] [Help]
Hello,
I am currently work with a international brand and want to implement a multilingual website. I want to integrate BigCommerce with Vercel Next.js. I wonder which part should handle the translation work? For each product, do I want to create different clones in my backend(BigCommerce) or do I want to use Next.js libraries like i18n and store the translation texts in the frontend?
Any suggestions would help, 
Thanks.

**Comment by mhdalmajid** [comment]
this link from official doc will be useful, have a look into it , if you need help ask again here plz 
https://nextjs.org/docs/advanced-features/i18n-routing
